widget/embedded/src/hooks/useConfirmSwap/useConfirmSwap.ts

```diff
@@ -39,7 +39,7 @@ export function useConfirmSwap(): ConfirmSwap {
 
   useEffect(() => cancelFetch, []);
 
-  const fetch: ConfirmSwap['fetch'] = async (params: Params) => {
+  const fetch: ConfirmSwap['fetch'] = async (params) => {
```

Comment:
I didn't understand the reason for this change.

Reply:
Typescript infers it automatically.